kubespawner/spawner.py

```diff
@@ -645,7 +734,8 @@ def _validate_image_pull_secrets(self, proposal):
         upgrades to break.
 
         You'll *have* to set this if you are using auto-provisioned volumes with most
-        cloud providers. See `fsGroup <https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#podsecuritycontext-v1-core>`_
+        #podsecuritycontext-v1-core>`_
```

Comment:
Same comment about the broken links. Looks like an auto-line-wrapper doesn't respect docstring contents.

Reply:
I'll patch these up as I implement the other suggestions.

Reply:
Done, generally got it up to referencing 1.20, tried to make the formatting consistent within a section.  Honestly refactoring all the docstrings is probably its own PR, to impose some kind of consistent style here, but...another day.